tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -298,6 +298,23 @@ def test_torch_large(self):
         output = speech_recognizer(filename)
         self.assertEqual(output, {"text": "A MAN SAID TO THE UNIVERSE SIR I EXIST"})
 
+    @require_torch
+    @slow
+    def test_torch_large_fbank(self):
+        speech_recognizer = pipeline(
+            task="automatic-speech-recognition",
+            model="hf-audio/wav2vec2-bert-CV16-en",
+            framework="pt",
+        )
+        waveform = np.tile(np.arange(1000, dtype=np.float32), 34)
```

Comment:
> it takes mel-spectrograms as input_features, instead of raw waveform as input_values

Is `wavefrom` the best name here? It looks a bit contradicted to the PR description.

Not a big deal, but if a more proper name is available, we can use it then.

Reply:
The input is a waveform but is processed into a melspectrogram (fbank) ! I'll keep this name

Reply:
On that note, I see docstring still says:
> input_features (`torch.FloatTensor` of shape `(batch_size, sequence_length)`):
            Float values of input raw speech waveform.

https://github.com/huggingface/transformers/blob/708b19eb093bcfb4efcfa229925b249397c2bd5e/src/transformers/models/wav2vec2_bert/modeling_wav2vec2_bert.py#L1020-L1025